routes/api/points.js

```diff
@@ -0,0 +1,16 @@
+const router = require("express").Router();
+const pointsController = require("../../controllers/pointsController");
+
+// Matches with "/api/books"
+router.route("/")
```

Comment:
This route will probably need to be updated to reference the different collections, instead of the root collection.